ndless-sdk/bin/nspire-tools

```diff
@@ -61,7 +81,17 @@ case "$1" in
 		cp "${DIRNAME}/main.c.tpl" main.c
 		;;
 	path)
-		echo "$(dirname "$DIRNAME")/"
+		echo "$NDLESS/"
+		;;
+	gcc)
+		GCC=$(toolchainbin)/arm-none-eabi-gcc
+		exec "$GCC" "$@"
+		;;
+	toolchainbin)
```

Comment:
I'd prefer `toolchainpath` for symmetry.

Additionally, it's "private" like `_zehnpath`, so `_toolchainpath`